heat/core/statistics.py

```diff
@@ -1439,7 +1439,7 @@ def percentile(
 ) -> DNDarray:
     r"""
     Compute the q-th percentile of the data along the specified axis.
```

Comment:
```suggestion
    Compute the q-th percentile of the data along the specified axis.
```
Compute the q-th percentile of the data along the specified axis/axes.